### DISCUSSION TITLE
Just playing around with it (ubuntu 20.04 rtx 3080, 32GB RAM, Razer Blade 2020)

### DISCUSSION THREAD

**Discussion by awesomebytes** [w4ffl35/krita_stable_diffusion#68] [Show and tell]
First, thanks for the plugin!
I enjoyed playing with it. It took me a moment to get it to work (I needed to build Krita from source, and then add that Krita bin folder to my PATH so I could install the plugin, I don't think it's possible to do it with flatpak/AppImage), but it is fun to play with. Ah, I also needed to create an empty image before starting to generate (I kept it to 512x512px as I read that it could be an okay size) or I won't see the results. Every image took about 5-10 seconds for me to generate. Also the upscaling did not work for me, there's an error about 'None' object when clicking on it.

I thought I'd share an image I generated just for fun:

![Screenshot from 2022-12-19 16-40-53](https://user-images.githubusercontent.com/1721716/208356025-19b79f95-fc09-480f-945b-cd4625f0ae28.png)

I also would liked to be able to just do img2img, so to modify generated images (if I understood correctly how it works) as I find that with text + image as input, it modifies the image a lot. For example my prompt contains a traffic light, and in most text+image generations from an image that already had the traffic light, the traffic light became a weird thing or disappeared.

Anyways, thanks again for the effort! It is very interesting to explore this new space.

**Comment by w4ffl35** [comment]
@awesomebytes thanks for the feedback, it means a lot.

I am aiming to release the next version tomorrow - it has txt2img, img2img, inpaint, outpaint and ability to use custom models. I'm just putting the finishing touches on it and fixing a few bugs. Please be sure to keep an eye out for the update, I think you'll enjoy it.

In the meantime, if you [follow me on mastodon](https://hachyderm.io/@w4ffl35) or [join our discord](https://discord.gg/PUVDDCJ7gz) or [follow me on twitter](https://twitter.com/xw4ffl35x) you will be able to get the latest updates.

**Comment by w4ffl35** [comment]
@awesomebytes i just saw you sponsored me - that's amazing! I appreciate this so much - you are my first sponsor.
My day wasn't going well - I'm behind on the release due to things in my personal life... then I saw your message and it made me feel better... but then I saw sponsorship as well, and that's just awesome. Thank you. I will do my best to get the next update out asap. I think you'll really like the new interface.

**Reply by awesomebytes** [reply]
Don't feel pressured to release anything faster! You've done a great job already, I got it working relatively quick, and got to play with it for a good hour, so I thought I may buy you at least a coffee!

Also, I don't know if I missed it anywhere, but in between building from source Krita + the plugin + the stable diffusion networks, I needed about 29GB of disk space free. It may be a good warning for the README.
